MIGRATION.md

```diff
@@ -0,0 +1,41 @@
+This document sums up issues for migrating between 0.1.4 to 0.9.x
```

Comment:
The guide is really clear, nice work 👍

Reply:
If there something else that needs to be documented let me know. It's just a start...